auth_supplier/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Auth Supplier
+=============
+
+This module was written to extends the functionality of auth signup
+and allow you to assign a supplier to a user when create an account.
```

Comment:
This module does not let *you* assign a supplier to an user, but allows *the user* to create an account as a supplier.